test/libsolutil/UTF8.cpp

```diff
@@ -150,7 +152,7 @@ hélló
 
 ₠ ₡ ₢ ₣ ₤ ₥
 
-⃐ ⃑ ⃒ ⃓ ⃔ ⃕ ⃖ ⃗ ⃘ ⃙ ⃚
```

Comment:
please undo

Reply:
that must have been an accident... or maybe my editor.. will revert